packages/api-client/src/rest/rest.ts

```diff
@@ -160,16 +195,23 @@ export class RestClientImpl implements RestClient {
 
   async tx (tx: Tx): Promise<TxResult> {
     const requestUrl = concatLink(this.endpoint, `/api/v1/tx/${this.workspace}`)
-    const response = await fetch(requestUrl, {
-      method: 'POST',
-      headers: this.jsonHeaders(),
-      keepalive: true,
-      body: JSON.stringify(tx)
-    })
-    if (!response.ok) {
-      throw new PlatformError(unknownError(response.statusText))
+    const result = await withRetry(async () => {
+      const response = await fetch(requestUrl, {
+        method: 'POST',
+        headers: this.jsonHeaders(),
+        keepalive: true,
```

Comment:
Should we add keep-alive headers to `this.jsonHeaders` as well?

Reply:
This is not this PR intend to check for it :-)